docs/Users/Onboarding.md

```diff
@@ -17,10 +17,10 @@ The first time you visit an app that uses Vincent for Authentication & Delegatio
 After you login you'll be required to review the permissions requested by the app on the Consent Page. On this page you can see the following details about the app:
 
     - **App:** The name and description of the app you're visiting.
-    - **Account Address:** The EOA wallet address of the App owner.
+    - **Account Address:** The EOA wallet address associated with the App creator.
     - **App Version:** The version of the app you're visiting. If the app publishes a new version you'll be prompted to update it.
-    - **Tools:** Tools are the operations that an App can perform on *behalf* of your Agent Wallet and these are codified as Lit Actions. Lit Actions are **immutable** Javascript code snippets assigned to your Agent's wallet. Thus once you delegate it these Tools to an App the App owner cannot change the Tool's behavior making the delegation system trustless. You always have an option to revoke the App's delegation after which they won't be able to perform actions on your behalf.
-    - **Policies:** Policies are the guard-rails for the permitted Tools to dictate the operating conditions for that Tool. Tools can have multiple Policies like max daily spend or 2FA and all these policies should be met before the Tool can sign using the delegated user's Agent wallet. You can configure the parameters of these Policies like specifying the max daily spend amount or the 2FA method. Just like the Tools, Policies are also codified as Lit Action.
+    - **Tools:** Tools are used to execute the operations that you permit a given app to perform on your *behalf* using your Agent Wallet. All Tools are represented by [Lit Actions](https://developer.litprotocol.com/sdk/serverless-signing/overview). After you delegate your Agent Wallet to a given Tool, the application owner can no longer change the Tool's behavior, making the delegation system trustless. As the owner of your Agent Wallet, you always have the ability to revoke a Tool that you previously permitted. 
```

Comment:
```suggestion
    - **Tools:** Tools are used to execute the operations that you permit a given app to perform on your *behalf* using your Agent Wallet. All Tools are represented by [Lit Actions](https://developer.litprotocol.com/sdk/serverless-signing/overview). After you permit a Tool to be used with your Agent Wallet for a specific app, the app owner can no longer change the Tool's behavior, making the delegation system trustless. As the owner of your Agent Wallet, you always have the ability to revoke a Tool that you previously permitted for any Vincent app. 
```

> making the delegation system trustless.

It's also not technically trustless, it's more appropriate to say it's immutable, but trustless does sound better